iam-login-service/src/main/java/it/infn/mw/iam/api/requests/GroupRequestsController.java

```diff
@@ -27,13 +32,13 @@
   private GroupRequestsService groupRequestService;
 
   @RequestMapping(method = RequestMethod.POST, value = {"", "/"})
-  @PreAuthorize("hasRole('ADMIN') or (hasRole('USER') and #iam.userCanCreateGroupRequest(#groupRequest))")
-  public GroupRequestDto createGroupRequest(@RequestBody GroupRequestDto groupRequest) {
+  @PreAuthorize("isAuthenticated()")
+  public GroupRequestDto createGroupRequest(@RequestBody @Valid GroupRequestDto groupRequest) {
     return groupRequestService.createGroupRequest(groupRequest);
   }
 
   @RequestMapping(method = RequestMethod.GET, value = "/")
-  @PreAuthorize("hasRole('ADMIN') or hasRole('USER')")
+  @PreAuthorize("isAuthenticated()")
```

Comment:
I think the previous annotation was more correct, since this annotation would allows access to OAuth authenticated clients.